FRAMEWORKS.yml

```diff
@@ -74,6 +74,10 @@ node:
     type: experimental
     language: 11.14
 python:
+  masonite:
+    language: "3.7"
+    version: "2.1.27"
```

Comment:
Could you use `X.`Y` format ?

Reply:
I could but will  miss the few updates.

Reply:
There will be tracked 😙

You can add a line on `.dependabot/config.yml`

Reply:
Okay, cool